extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -467,7 +467,10 @@ export class AmpConsent extends AMP.BaseElement {
           return Promise.resolve(consentRequired);
         }
         return this.getConsentRemote_().then(consentResponse => {
-          return !!consentResponse['consentRequired'];
+          // `promptIfUnknown` is a legacy field
+          return consentResponse['consentRequired'] !== undefined
+            ? !!consentResponse['consentRequired']
+            : !!consentResponse['promptIfUnknown'];
```

Comment:
Please add a comment and also document in readme later, that when a response cannot be found, amp-consent will ignore the prompt and unblock components. Publishers need to be extra careful with this.

Reply:
And we can go ahead and remove legacy method in the next PR : )